parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/AbstractParallelEoSStreamProcessor.java

```diff
@@ -678,23 +695,26 @@ private void doClose(Duration timeout) throws TimeoutException, ExecutionExcepti
         if (Thread.currentThread().isInterrupted()) {
             log.warn("control thread interrupted - may lead to issues with transactional commit lock acquisition");
         }
-        commitOffsetsThatAreReady();
-
+        try {
+            commitOffsetsThatAreReady();
+        } catch (Exception e) {
+            log.error("failed to commit during close sequence", e);
```

Comment:
I think warning is more warranted here? there should be already error logged that lead to close() invocation.
```suggestion
            log.warn("failed to commit during close sequence", e);
```